apps/store/extensions/app/social-reviews/modules/review-api.js

```diff
@@ -29,6 +29,7 @@ var api = {};
     var HTTP_GET_METHOD = 'GET';
     var HTTP_POST_METHOD = 'POST';
     var HTTP_DELETE_METHOD = 'DELETE';
+    var HTTP_PUT_METHOD = 'PUT';
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/536cfb02b4ebbf5e1f2a5366870034af2b7b9616)